[33mWARNING [0m django.request:log.py:253 Not Found: /api/user/auth/login/
[1m[31mERROR   [0m error:knowledge_management.py:221 创建文档验证失败: {'form_data': {'fields': [ErrorDetail(string='字段名不能重复', code='invalid')]}}
[33mWARNING [0m django.request:log.py:253 Bad Request: /knowledge/namespaces/8/documents/